### ISSUE TITLE
Report when Session client is outdated

### ISSUE THREAD

**Issue by erdh** [oxen-io/session-android#947]
While chatting with my friend who somewhat didn't updated their Session client, which fixed a bug about disappearing messages on iOS platform, (patch 2.0.1) which made the chats that are intended to disappear weren't disappeared and caused a security panic for both end. If the other end or you are not in charge that client is not running the latest patches that fixes important issues, might cause big problems when critical bugs are patched.

Could be replicated with just chatting with someone that doesn't have the latest patch.

If the Session app would just give a message that this client is not the latest patch, might fix issues quicker because the user is now warned.

**Comment by hjubb** [comment]
The main concern that would come alongside this is that you will know or reveal to the people you are chatting with what platform and version of the application you are using, which can be used to identify you. It could be worthwhile to know on your own device if there is an updated, however it would then still be on the individual to do anything about it

**Comment by erdh** [comment]
Yes, on second thought, depends on the threat model, giving info that the other hand is not using latest patch might reveal a bit info about them. If the Session app would give a message to client that they are not using the latest patch, (of course we can not force them) that will possibly encourage them to update their client for safer chat for both ends.

**Comment by KeeJef** [comment]
Yeah i think this would be cool, we already have a similar feature in Session Desktop which reports if your version of Session is out of date, would be cool to do the same for mobile versions, would be useful for people who directly download the APK version.
